### ISSUE TITLE
`/en/latest` prefix still added with `notfound_no_urls_prefix = True` set

### ISSUE THREAD

**Issue by acozine** [rtfd/sphinx-notfound-page#16]
@humitos @banesullivan thank you for writing a plugin for custom 404 pages in sphinx. We'd like to use it in our documentation.

Our docs URLs look like `https://docs.ourdomain.com/project_name/version_name/page.html`. I added the `notfound-page` extension and set `notfound_no_urls_prefix = True` in my Sphinx `conf.py`,  but in the html output (`404.html`) I still see paths with the default `en/latest` in the path:

```
<link rel="stylesheet" href="/en/latest/_static/css/theme.css" type="text/css" />
<link rel="stylesheet" href="/en/latest/_static/pygments.css" type="text/css" />
<link rel="index" title="Index" href="/en/latest/genindex.html" />
<link rel="search" title="Search" href="/en/latest/search.html" /> 
```

Is this a bug, or am I misunderstanding/misusing the extension?

I'm a technical writer, not a python dev, but I'd be happy to get involved if I can help get this project to a 1.0 release.

**Comment by banesullivan** [comment]
Thanks for reporting your issue! Is this project open-source where I could look at the way the docs are set up for further insight?

**Comment by acozine** [comment]
@banesullivan thanks for the quick response! Here's the branch where I'm trying to use `sphinx-notfound-page`: https://github.com/acozine/ansible/tree/custom-404

**Comment by acozine** [comment]
and yeah, it's open-source

**Comment by banesullivan** [comment]
Just to make sure, you are using the latest release of this package, right? Perhaps try installing from the master branch to confirm?

`pip install git+git://github.com/rtfd/sphinx-notfound-page@master`

**Comment by acozine** [comment]
I'll try pip install from git, but for now:
```
[alicia@blah]~/path/$ pip show sphinx-notfound-page
Name: sphinx-notfound-page
Version: 0.2.1
Summary: Sphinx extension to build a 404 page with absolute URLs
Home-page: https://github.com/rtfd/sphinx-notfound-page
Author: Manuel Kaufmann
Author-email: humitos@gmail.com
License: UNKNOWN
Location: /Library/Python/2.7/site-packages
Requires: sphinx
Required-by: 
```

**Comment by acozine** [comment]
Heading out for the long weekend. I'll pick this up next week. @banesullivan if you're open to looking at this in real time, ping me on the #ansible-docs channel on freenode IRC, Tuesday or after.

**Comment by humitos** [comment]
@acozine I think @banesullivan is right. You may not be installing the version from the git repository but from PyPI (where the latest version released does not include the config option that you need).

> Update: when I try `pip install git+. . . .` I get `Requirement already satisfied`

Try doing `pip uninstall sphinx-notfound-page` and then `pip install git+...`. It should work.

Let me know otherwise.

**Comment by humitos** [comment]
I just released the version 0.3 that includes this feature and I created better docs for this project.

You should be able to install it from pip now using the `pip install -U sphinx-notfound-page` now.

**Comment by acozine** [comment]
Confirmed that the 0.3 release fixed my problem. Thanks @humitos @banesullivan.
